client/src/App.tsx

```diff
@@ -57,6 +58,9 @@ function App() {
 	const [musicNotes, setMusicNotes] = useState<IMusicNoteProps[]>([]);
 	const [emojis, setEmojis] = useState<IEmoji[]>([]);
 	const [gifs, setGifs] = useState<IGifs[]>([]);
+	const [backgroundName, setBackgroundName] = useState<string | undefined>(
```

Comment:
the default state should be undefined not 'none', 'none' has the same issue as 'blank' 'empty' etc in that it's unclear compared to undefined